File: Client/app/shared/patient.ts
import { PatientDiagnosis } from './patientDiagnosis'
export class Patient{

    id: number;
    name: string;
    dateOfBirth: Date;
    gender: Gender;
    weight: number;
    height: number;
    race: Race;
    inclusionDate: Date;
    general: string;
    diagnosis:PatientDiagnosis[];

    fromJSON(json:Object) {
        for (var propName in json)
            this[propName] = json[propName];
        return this;
    }
}

export enum Gender{
    Male,
    Female
}
export enum Race{
    Caucasian,
    Black,
    Latino,
    Asian,
    Other
}

File: Server/MRP.DAL/Models/Patient.cs
using MRP.Common.DTO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MRP.DAL.Models
{
    public class Patient
    {
        public string Id { get; set; }
        public int PatientId { get; set; }
        public string Name { get; set; }
        public DateTime DateOfBirth { get; set; }
        public Gender Gender { get; set; }
        public int Weight { get; set; }
        public int Height { get; set; }
        public Race Race { get; set; }
        public DateTime InclusionDate { get; set; }
        public string General { get; set; }
        public DateTime LastModified { get; set; }
        public IEnumerable<PatientDiagnosis> Diagnosis { get; set; }
    }
}
